logging_format/visitor.py

```diff
@@ -238,6 +238,22 @@ def get_except_handler_name(self, node):
             return name.id
         return name
 
+    def get_id_attr(self, value, attempt=1):
```

Comment:
It's not obvious to me at all why this function needed to change or, especially, why it needs to be recursive.

I'd appreciate:
 - At least one unit test for the scenario you are trying to address in this PR.
 - Either additional justification for this complexity (e.g. more tests or doc strings) or a much simpler function that narrowly addresses your test case.

Reply:
The log is missed if 'level' more that max attempt value.
The test represents the scenario in which this occurs.

Reply:
Thank you for the test case; it clarifies somewhat.

I'm still a little unsure about that this syntax is a reasonable thing for a general purpose linting library to do. Most code that I've encountered would look more like:

```
class Child:
    def __init__(self:
        self.logger = getLogger("Child")

    def bar(self):
        self.app.log.info("whatever")

app.child = Child()
```

And then would trigger logging via:

```
app.child.bar()
```

rather than "reaching into" `app.child.logger` directly. That is, if code respects separation of concerns, the problem you're solving here shouldn't be necessary.

Can you perhaps give me a little more context/justification around why this code style is important to you?

Reply:
The flask and chalice frameworks use `app.log` for logging. flake8-logging-format failed to detect this when linting logging calls. When testing I found that it worked with 5 and this felt like a good number for making the code flexible.

Upon reflection 2 is appropriate. Anything high endorses bad coding practices. I will update my code tomorrow.

Thanks for the reviews.